docs/CHANGELOG.asciidoc

```diff
@@ -73,6 +73,7 @@ The 8.13.2 release includes the following bug fixes.
 === Bug Fixes
 Canvas::
 * Fixes text settings to be honored in Canvas markdown elements ({kibana-pull}179948[#179948]).
+* Fixes custom styling for Canvas Workpads ({kibana-pull}180070[#180070]).
```

Comment:
Same here, better to specify Markdown, because the changes are only related to markdown CSS